cmr_logs_categorizer/export_logs.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+# Function to display usage
+show_usage() {
+    echo "This script exports CMR logs for the specified date range to a S3 bucket."
+    echo
+    echo "Ensure the dates are formatted according to the examples below."
+    echo "The dates are inclusive: StartDate begins at midnight, and the"
+    echo "EndDate ends at 11:59 PM."
+    echo
+    echo "Usage:   $0 StartDate EndDate"
+    echo "Example: $0 \"Oct 08 2024\" \"Oct 15 2024\""
+    echo
+    echo "Optionally provide the Cloudwatch log group name (default: cmr-search-prod)"
+    echo "AWS profile name (default: cmr-prod), AWS region (defualt us-east-1),"
+    echo "and destination bucket (default: bigstac-cmr-prod-logs):"
+    echo "Usage:   $0 StartDate EndDate LOG_GROUP_NAME AWS_PROFILE AWS_REGION S3_BUCKET"
+}
+
+# Check if no arguments were provided
+if [ $# -eq 0 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
```

Comment:
have you ever tried getups? 

`opt="default"
while getopts "ho:" OPTION; do
    case $OPTION in
    h) help_doc ; exit ;;
    o) opt=$OPTARG" ;;
    esac
done`